.github/ISSUE_TEMPLATE/--feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "Feature request"
+about: Request/suggest a new feature in VW
+title: ''
+labels: "Feature Request"
+assignees: ''
+
+---
+
+<!-- Please check if there is an existing issue that matches your request. -->
+
+## Short description
+<!-- Describe your idea -->
+
+
+## **How this feature will help you/others**
+
+
+## **Possible solution/implementation details**
+<!-- Describe the solution you'd like. -->
+
+
+## **Example/links if any**
+<!-- Link to already similar features if possible -->
+
+## **Can You Help With This Feature**
```

Comment:
```suggestion
```

I don't think this is needed. It is already clear the contributions are welcome and I don't want someone to refrain from opening a feature request because they think they need to implement it